tests/lib/common.py

```diff
@@ -67,11 +67,11 @@ def wait_for_result(func, *args, matcher=simple_matcher(True), attempts=20,
     raise Exception("Timed out waiting for result")
```

Comment:
Commit message `Add support for disabling logging to execute util`: can you explain why this is useful please?

Reply:
There may be some commands that are too verbose on stdout. Disabling the logging means you can do your own logging or capture the output and log parts of it etc.